test/unit-tests/debugger/debugAdapterFactory.test.ts

```diff
@@ -45,18 +45,18 @@ suite("Debug Adapter Factory Test Suite", () => {
             const configProvider = new LLDBDebugConfigurationProvider("darwin", swift6);
             const launchConfig = await configProvider.resolveDebugConfiguration(undefined, {
                 name: "Test Launch Config",
-                type: DebugAdapter.adapterName,
+                type: "swift-lldb",
```

Comment:
Probably good idea to the use the newly added enum here as well?

Reply:
Yup. Good catch!